modules/network-observability-dns-tracking.adoc

```diff
@@ -36,7 +36,7 @@ spec:
       sampling: 1              <2>
 ----
 <1> You can set the `spec.agent.ebpf.features` parameter list to enable DNS tracking of each network flow in the web console.
-<2> You can set `sampling` to a value of `1` for more accurate metrics.
+<2> You can set `sampling` to a value of `1` for more accurate metrics and to capture *DNS latency*. For a `sampling` value greater than 1, you can observe flows with *DNS Response Code* and *DNS Id*, and while unlikely, there can be observed *DNS Latency*.
```

Comment:
wording feels off to me, better to write
`and unlikely that DNS Latency can be observed`

Reply:
Yea :/ I was unsure tbh if I should even mention it since its unlikely. Addressed.